gointelowl/gointelowl.go

```diff
@@ -0,0 +1,97 @@
+package gointelowl
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+// Struct for IntelOwl.
+// Initiate it using Token, Instance, Certificate
+type IntelOwl struct {
+	Token       string
+	Instance    string
+	Certificate string
+}
+
+// Fetch list of all tags.
+//        Endpoint: ``/api/tags``
+// It Returns Slice of Tags
+func (intelOwl IntelOwl) GetAllTags() []map[string]string {
+	url := intelOwl.Instance + "/api/tags"
+	request, err := http.NewRequest("GET", url, nil)
```

Comment:
Line 23-34 are similar in all these methods, so we should extract this into an utility function like `buildAndMakeRequest`.